test/fixtures/posts.yml

```diff
@@ -480,3 +480,18 @@ question_without_votes:
   community: sample
   category: main
   license: cc_by_sa
+
+post_scorer_question:
+  post_type: question
+  title: This is a question for testing post scores
+  body: This post MUST be positively scored to count towards the score
```

Comment:
World's tiniest nit: for the benefit of future readers, maybe don't say "positively scored" when we mean > 0.5?  0.3 is positive mathematically but not what you meant.